### ISSUE TITLE
When creating a RowRenderer with a factory, the control is not editable

### ISSUE THREAD

**Issue by benshoe** [fjalvingh/domui#18]
Example:
```java
RowRenderer<SomeClass> rr = new RowRenderer(SomeClass.class);
rr.column(BigDecimal.class, "property").factory(row -> {
Text<BigDecimal> ctrl = new Text<>(BigDecimal.class);
ctrl.bind().to(row, "property");
return ctrl;
}
).editable();
```
Without the 'editable()' method, the control is shown in read-only mode.

**Comment by benshoe** [comment]
Hi, would this be a valid unit test for this bug?

public class RowRendererTest {

	public class TestObject {

		BigDecimal m_property;

		public BigDecimal getProperty() {
			return m_property;
		}

		public void setProperty(BigDecimal property) {
			m_property = property;
		}
	}

	@Test
	public void testEditableWithFactory() {
		RowRenderer<TestObject> rr = new RowRenderer<>(TestObject.class);
		rr.column(BigDecimal.class, "property").factory(row -> {
				Text2<BigDecimal> ctrl = new Text2<>(BigDecimal.class);
				ctrl.bind().to(row, "property");
				return ctrl;
			}
		).editable();

		ColumnDef<TestObject, ?> column = rr.getColumn(0);
		Assert.assertTrue("Column is editable", column.isEditable());

		rr.column(BigDecimal.class, "property").factory(row -> {
				Text2<BigDecimal> ctrl = new Text2<>(BigDecimal.class);
				ctrl.bind().to(row, "property");
				return ctrl;
			}
		);

		column = rr.getColumn(1);
		Assert.assertTrue("Column is also editable", column.isEditable());
	}
}

**Comment by fjalvingh** [comment]
No, the second part is incorrect. You would have to validate that, after a render, the _control_ that was rendered was editable. In this case you're just checking that the variable inside the column definition has been set - both your checks are the same.

You can either create a Selenium test (not that difficult, there is lots of infra) or make a test calling renderRow() of the RowRenderer. You would then get, inside the ColumnContainer, a call to add that control that was created. You can add a check for that.

**Comment by benshoe** [comment]
I have created a pull request for this issue.
